src/v/kafka/server/handlers/describe_configs.cc

```diff
@@ -553,13 +554,19 @@ ss::future<response_ptr> describe_configs_handler::handle(
 
             // Data-policy property
             ss::sstring property_name = "redpanda.datapolicy";
+            auto& v8_api = ctx.v8_api();
```

Comment:
can you elaborate more in commit message why the change was needed. Also this change is not related with the protocol but with the protocol handlers.